addons/website/views/snippets/snippets.xml

```diff
@@ -121,6 +121,27 @@
     </xpath>
 </template>
 
+<template id="snippet_options_background_options" inherit_id="web_editor.snippet_options_background_options">
+    <xpath expr="//we-button[@data-toggle-bg-image]" position="after">
+        <we-button title="Video" data-toggle-bg-video="true" data-no-preview="true" data-name="bg_video_toggler_opt">
+            <i class="fa fa-fw fa-film"/>
+        </we-button>
+    </xpath>
+    <xpath expr="//div[@data-js='BackgroundOptimize']" position="after">
+        <div data-js="BackgroundVideo"
+             t-att-data-selector="selector"
+             t-att-data-exclude="exclude"
+             t-att-data-target="target">
```

Comment:
I think there is a missing `t-att-data-dependencies="images_dependencies"` here (the dependencies that was there for the picker which allowed both image and video)